kiwi/filesystem/clicfs.py

```diff
@@ -82,7 +87,7 @@ def create_on_file(self, filename, label=None, exclude=None):
             ['mkclicfs', clicfs_container_filesystem, filename]
         )
 
-    def _get_container_filesystem_size_mbytes(self):
+    def _get_container_filesystem_size_mbytes(self) -> int:
```

Comment:
not a correction/nitpicking, question for me:
I understood the methods for the strong typing were the API ones,
are helpers included too ?

Reply:
My bad on this one, I was just adding type blindly to every method. You are right no need to add it here.